src/ReinforcementLearningZoo/src/algorithms/nfsp/nfsp_example.jl

```diff
@@ -0,0 +1,140 @@
+export NFSPAgent
```

Comment:
Just name this file as `nfsp.jl` ?

Reply:
Thanks for your kind suggestions!

I'll merge `nfsp_example.jl` and `nfsp_extensions.jl` to `nfsp.jl`, and move NFSPAgent setting details to the experiment file later.

> Please remember to fix the failed CI first.

I'll fix the spellcheck later. And I don't know how to fix the Documentation error. 😞

Reply:
Use `##` in the comments of `JuliaRL_NFSP_KuhnPoker.md` file instead of `#`.

Reply:
Thanks! I'll update the file soon.